src/commands/deploy/latest.js

```diff
@@ -281,10 +298,18 @@ export default async function main(
         meta
       },
       {
-        name: project
+        name: project,
+        target: argv['--target']
       }
     );
 
+    if (createArgs.target && createArgs.target !== 'production') {
+      error(`The specified ${highlight('target')} ${
```

Comment:
Let's use the correct output function for this, I think it's `param` and then `--target` inside or something. Please check the code.